### PR TITLE
Added login page and Protected Routes

### PR DESCRIPTION
**Pull request by steven-steven**
1. Create Login Page (📸 below)
2. Use dummy state 'isAuthenticated' and implement following behaviors:
   - All protected routes will redirect to /login if this state is false. (currently it resets to false every time you hard refresh, and set to true once you click 'Login')
   - A `/login` component will redirect to `/applications` if this is true.
3. Moved Navigation, Headers, Footer and put it all under 'Container'. Renamed 'Container' component into `PageLayout.jsx` and wrap this component around every main pages (Home, ApplicationsTable, Application, Comparison). I did this because not all routes (ie. /login) will need these layout wrappers.

loginPage1
loginPage2
loginPage3

### CODE REVIEW

File: src/Components/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    color: black;
+  }
+`;
+
+export default function Login({login, logout, isAuthenticated}) {
+
+  const errorEmailMessage = "Couldn't find your account";
+  const errorPasswordMessage = "Wrong password. Try again or click Forgot password";
+  
+  const [values, setValues] = useState({
```
Comment (GodGreg) [inline src/Components/Login/Login.jsx]: good :)

File: src/App.js [hunk 2]
```diff
@@ -127,37 +134,41 @@ export default class App extends Component {
         )
     }
 
+    const PrivateRoute = ({component: Component, ...rest }) => {
```
Comment (GodGreg) [inline src/App.js]: I added this part in my PR and created a separate file for the routes. When #36 is merged I will merge this one

File: src/Components/Login/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  function handleSubmit(event) {
+    event.preventDefault();
+    login();
+  }
+
+  const handleChange = prop => event => {
+    setValues({ ...values, [prop]: event.target.value });
+  };
+
+  if(isAuthenticated){
```
Comment (GodGreg) [inline src/Components/Login/Login.jsx]: isAuthenticated will be given by a context defined higher up :). I will change this later

File: src/Components/PageLayout/PageLayout.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
```
Comment (GodGreg) [inline src/Components/PageLayout/PageLayout.jsx]: Designs look good

The only thing that needs to be changed is the email/password title locations

image

Comment (steven-steven) [reply src/Components/PageLayout/PageLayout.jsx]: Ok. The design have no label so I wasn't sure what would happen if the fields are populated. Maybe have this label but position it a bit up?

**Pull request merged**